zookeeper-server/src/test/java/org/apache/zookeeper/test/ClientSSLTest.java

```diff
@@ -22,7 +22,9 @@
 
 package org.apache.zookeeper.test;
 
+import java.io.IOException;
 import static org.junit.Assert.assertTrue;
```

Comment:
checkstyle don't like the order for this... yikes. 
I guess Assert.fail should be before? Not sure why.

Reply:
oh, nevermind, static imports should be first, that's the problem.